openrtb_ext/response.go

```diff
@@ -96,3 +100,33 @@ const (
 	UserSyncIframe UserSyncType = "iframe"
 	UserSyncPixel  UserSyncType = "pixel"
 )
+
+// Bid is Wrapper around original/proxy bid object
+type Bid struct {
+	openrtb2.Bid
+	ID    string `json:"id,omitempty"`    // added omitempty
+	ImpID string `json:"impid,omitempty"` // added omitempty
+
+	OriginalBidCPM float64 `json:"originalbidcpm,omitempty"`
+	OriginalBidCur string  `json:"originalbidcur,omitempty"`
+}
+
+type Prebid struct {
+	Bid Bid `json:"bid"`
+}
```

Comment:
`Prebid` is too generic of a name for this package. Please name it `ExtResponseNonBidPrebid` to keep with the established naming convention.

Reply:
Done